notes/prepare-for-1.6.0.md

```diff
@@ -0,0 +1,55 @@
+# Prepare for Azure Management Libraries for Java 1.6.0 #
+
+Steps to migrate code that uses Azure Management Libraries for Java from 1.5 to 1.6 ...
+
+> If this note missed any breaking changes, please open a pull request.
+
+
+V1.6 is backwards compatible with V1.5 in the APIs intended for public use that reached the general availability (stable) stage in V1.x. 
+
+Some breaking changes were introduced in APIs that were still in Beta in V1.5, as indicated by the `@Beta` annotation.
+
+
```

Comment:
we should remove the following sections as there is no `Renames/Removals` or ` Changes in Input Parameters Order`

Reply:
Dropped